### ISSUE TITLE
Encountered “Minified React error” in the production environment

### ISSUE THREAD

**Issue by Zasvieeel** [jonluca/vite-typescript-ssr-react#12]
Hello. I've hosting this app on vercel and got unexpected error in console

Are this refers to vite configuration or not?

![image](https://user-images.githubusercontent.com/88622190/183597763-9d8ab21e-4b08-44e5-b382-a7454c939bd4.png)

Please help..

**Comment by jonluca** [comment]
That error is https://reactjs.org/docs/error-decoder.html/?invariant=423&args%5B%5D=ni

There was an error hydrating the page - it means that the html that was shipped down the page was not SSRd.

This commit should fix it https://github.com/jonluca/vite-typescript-ssr-react/commit/63c142ec9608155e61c2729944198c07fe63ee17

But the relevant change is in `entry-client.tsx`

```typescript
if (import.meta.hot || !container?.innerText) {
  const root = createRoot(container!);
  root.render(<FullApp />);
} else {
  hydrateRoot(container!, <FullApp />);
}
```
